responses/fr/HassGetWeather.yaml

```diff
@@ -3,22 +3,200 @@ responses:
+            {% set wind_scale = 'Petite brise' %}
+          {% elif wind_speed <= 28 %}
+            {% set wind_scale = 'Jolie brise' %}
+          {% elif wind_speed <= 38 %}
+            {% set wind_scale = 'Bonne brise' %}
+          {% elif wind_speed <= 49 %}
+            {% set wind_scale = 'Forte brise' %}
+          {% elif wind_speed <= 61 %}
+            {% set wind_scale = 'Grosse brise' %}
+          {% elif wind_speed <= 74 %}
+            {% set wind_scale = 'Coup de vent' %}
+          {% elif wind_speed <= 88 %}
+            {% set wind_scale = 'Fort coup de vent' %}
+          {% elif wind_speed <= 102 %}
+            {% set wind_scale = 'Tempête' %}
+          {% elif wind_speed <= 117 %}
+            {% set wind_scale = 'Violente tempête' %}
+          {% else %}
+            {% set wind_scale = "vent d'ouragan" %}
+          {% endif %}
+        {% elif wind_speed %}
+          {# Other wind speed unit, no scale, just the speed #}
+          {% set wind_scale = 'Vent' %}
+        {% endif %}
+
+        {% set wind_bearing_text = '' %}
+        {% if wind_bearing %}
+          {% if wind_bearing < 45 %}
+            {% set wind_bearing_text = 'nord' %}
+          {% elif wind_bearing < 90 %}
+            {% set wind_bearing_text = 'nord-est' %}
+          {% elif wind_bearing < 135 %}
+            {% set wind_bearing_text = 'est' %}
+          {% elif wind_bearing < 180 %}
+            {% set wind_bearing_text = 'sud-est' %}
+          {% elif wind_bearing < 225 %}
+            {% set wind_bearing_text = 'sud' %}
+          {% elif wind_bearing < 270 %}
+            {% set wind_bearing_text = 'sud-ouest' %}
+          {% elif wind_bearing < 315 %}
+            {% set wind_bearing_text = 'ouest' %}
+          {% elif wind_bearing <= 360 %}
+            {% set wind_bearing_text = 'nord-ouest' %}
+          {% else %}
+            {% set wind_bearing_text = 'inconnue' %}
+          {% endif %}
+        {% endif %}
+
+        {% if wind_speed_unit == 'km/h' %}
+          {% set wind_speed_unit = 'kilomètres par heure' %}
+        {% endif %}
+
+        {% if wind_speed and wind_bearing and wind_gust_speed %}
+          {{ wind_scale }} de {{ wind_speed | int }} {{ wind_speed_unit }}, dont la direction est {{ wind_bearing_text }}, avec des rafales à {{ wind_gust_speed | int }}.
+        {% elif wind_speed and wind_bearing %}
+          {{ wind_scale }} de {{ wind_speed | int }} {{ wind_speed_unit }}, dont la direction est {{ wind_bearing_text }}.
+        {% elif wind_speed and wind_gust_speed %}
+          {{ wind_scale }} de {{ wind_speed | int }} {{ wind_speed_unit }}, avec des rafales à {{ wind_gust_speed }}.
+        {% elif wind_speed %}
+          {{ wind_scale }} de {{ wind_speed | int }} {{ wind_speed_unit }}.
+        {% else %}
+          Aucune information météo sur le vent n'est disponible.
+        {% endif %}
+
+      uv: >
+        {% set uv_index = state.attributes.get('uv_index') %}
+
+        {% if uv_index %}
```

Comment:
Does this particular scale have a name?
I quite like the comment on the Beaufort scale for the wind.

Reply:
This is the scale I found on the Météo-France website. I could leave a link as the source, but I have no idea how much of a standard it is.